src/components/StructuredContent/StructuredContent.stories.tsx

```diff
@@ -122,17 +122,17 @@ export const WithHTMLStringTextContent: Story = {
         "eiusmod tempor incididunt ut labore et dolore magna aliqua. " +
         "Pellentesque massa placerat duis ultricies lacus sed turpis tincidunt. " +
         "<a href='#'>Porttitor eget dolor</a> morbi non arcu risus quis varius. " +
-        "Faucibus in ornare quam viverra orci sagittis.</p><p>Nisl vel pretium " +
+        "Faucibus in ornare quam viverra orci sagittis.</p><p style='margin-top: 1rem'>Nisl vel pretium " +
```

Comment:
Can we target the `p` element when it's within the `StructureContent` component?